nilearn/glm/tests/test_thresholding.py

```diff
@@ -37,6 +33,19 @@ def test_fdr(rng):
     assert np.isfinite(fdr_threshold(norm.isf(pvals), 0.1))
 
 
+def test_fdr_error(rng):
+    n = 100
+    x = np.linspace(0.5 / n, 1.0 - 0.5 / n, n)
+    x[:10] = 0.0005
+    x = norm.isf(x)
+    rng.shuffle(x)
+
+    with pytest.raises(ValueError):
+        fdr_threshold(x, -0.1)
+    with pytest.raises(ValueError):
```

Comment:
Want to make the error explicit ?

Reply:
good point will actually do that for all the "bare" `pytest.raises`

Reply:
This makes me realize that bare pytest.raises were not catching we would expect from the comments in the test or from the logic of the test.

We should probably go through the test suite to ensure that error and warning messages are properly tested.